tests/end2end/features/test_fileselect_bdd.py

```diff
@@ -0,0 +1,62 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2016-2020 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
```

Comment:
```suggestion
# Copyright 2020 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
```

Probably irrelevant with the comment above though (and I should really update those for 2021...)

Reply:
Yep, since merged with editor tests